sdk/mgmtcommon/AppAuthentication/Azure.Services.AppAuthentication/AccessToken.cs

```diff
@@ -99,6 +99,24 @@ public override string ToString()
             return _accessToken;
         }
 
+        /// <summary>
+        /// Return the deserialized tenantId if accessible
+        /// </summary>
+        /// /// <param name="accessToken">AccessToken to deserialize.</param>
+        /// <returns></returns>
```

Comment:
Should we consider filling this out or removing it?

Reply:
I copied the format used in other files in this project ie: https://github.com/Azure/azure-sdk-for-net/blob/6052a04aa67333828d3c882fe550ee1f2fc9ed40/sdk/mgmtcommon/AppAuthentication/Azure.Services.AppAuthentication/TokenProviders/ClientCertificateAccessTokenProvider.cs#L121

Reply:
The type is internal, so not a big concern.